src/graph/service/GraphService.cpp

```diff
@@ -80,7 +80,7 @@ folly::Future<AuthResponse> GraphService::future_authenticate(const std::string&
     return future;
   }
 
-  if (!sessionManager_->isWithinConnections()) {
+  if (!sessionManager_->isOutOfConnections()) {
```

Comment:
You should keep `isOutOfConnections` fit  its name.

Reply:
yixinglu suggests isOutOfConnections  prefer.

Reply:
The name of this function is obviously conflict with its implementation.

Reply:
Obviously, it should be:
```
if (isOutOfConnections) {
//report error
}
```